config.js

```diff
@@ -32,6 +33,16 @@ export function Config(onClose) {
         this.model.tube = enabled ? findModel("Tube65c02") : null;
     };
 
+    this.setEconet = function (enabled) {
+        enabled = !!enabled;
+        $("#hasEconet").prop("checked", enabled);
+        this.model.hasEconet = enabled;
+
+        if (enabled && this.model.name.toString().includes("Master")) {
```

Comment:
I'd rather make "is it a master" more explicit than the string check here: though this is OK for now

Reply:
Oh! There's already an `isMaster` :) will change